src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DepositOrderValidationService.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import org.apache.commons.collections4.CollectionUtils;
+import uk.gov.hmcts.ecm.common.model.ccd.CaseData;
+import uk.gov.hmcts.ecm.common.model.ccd.items.DepositTypeItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.CLAIMANT_TITLE;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.DEPOSIT_REFUNDED_GREATER_DEPOSIT_ERROR;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.RESPONDENT_TITLE;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.UNABLE_TO_FIND_PARTY;
+
+public class DepositOrderValidationService {
+
+    private DepositOrderValidationService() {
+    }
+
+    public static List<String> validateDepositOrder(CaseData caseData) {
+        List<String> errors = new ArrayList<>();
+        if (!CollectionUtils.isEmpty(caseData.getDepositCollection())) {
+
+            for (DepositTypeItem depositTypeItem : caseData.getDepositCollection()) {
+                if (!isNullOrEmpty(depositTypeItem.getValue().getDepositAmountRefunded())
```

Comment:
Put this in a method to follow the pattern below it. Also, the method name should clearly express what it is validating as the if statement is quite hard to follow

Reply:
Extracted